src/fetchState/replaceState.ts

```diff
@@ -0,0 +1,32 @@
+import { AnyAction, Reducer } from 'redux'
+import { FluxStandardAction } from '../utils/isFSA'
+import { ActionMeta } from '../utils'
+
+interface ReplaceStateAction<S> extends FluxStandardAction<ActionMeta> {
+    payload: S
+}
+
+const REPLACE_STATE = 'electron-redux.REPLACE_STATE'
```

Comment:
I would prefer to have all actions at one place but it's also fine like this.

Reply:
This is action type that is redux specific, and its scoped only to this module, while others are grouped as ipc events _topic_ or _channels_, hence the separation